fendermint/vm/topdown/src/sync.rs

```diff
@@ -22,6 +22,15 @@ const MAX_PARENT_VIEW_BLOCK_GAP: BlockHeight = 100;
 /// When polling parent view, if the number of top down messages exceeds this limit,
 /// the polling will stop for this iteration and commit the result to cache.
 const TOPDOWN_MSG_LEN_THRESHOLD: usize = 500;
+/// The null round error message
+const NULL_ROUND_ERR_MSG: &str = "(code: 1, message: requested epoch was a null round, data: None)";
```

Comment:
I wouldn't make it this specific. What if they add some data or change the code? As long as `"requested epoch was a null round"` is present I don't think the other details such as tuple formatting are needed.

Reply:
Agree, let's have a regexp. An example of how things can go south if we use matching strings: 
* This is what returns the error: https://github.com/filecoin-project/lotus/blob/master/node/impl/full/eth_utils.go#L28
* But some function may add additional context info at the beginning of the error :face_exhaling: https://github.com/filecoin-project/lotus/blob/e8455567fd6a0d3e590f0f2b695b765526637954/node/impl/full/eth.go#L701-L703

It is not the case for the call we are using, but that way we are "safer" in the future.